src/routes/blog/[id$]/My Neck, My Back, My Updates and My App .md

```diff
@@ -0,0 +1,253 @@
BFp9Otw1WcFrcziv+8rGbk5PW+0hnZHGDla7OjPFODjymUNvtXaumxbOI05rsINHwkjzrlcRMGXdpKit0mlRvALBiG0LYbmGw4aLK6PPg2AqKlZ3DFD0mK0dnY0qn47d7z9XuOaaP3EHjD3/fYU0ZCBsBPnYpqlyMvhxcZqSlAfCMDtLIvl2Pyivha1LF5Ysczx2koclkkomRNUz2AmnKWIK1zbotqyeFbeFfiPP9Mc6vTgewr8NQz93qFdvz7LYsh45WRxAdsWzDY9gJcijJ8q+E8zHQjupNy+QSBxinv9kFh8/pgNf4nMUtpHut3IBXUR9hB9aRr5upuvVXAi6lcyKTf4EfX6keJZv+QJ+GT31eceBafRdIhlV02rrd2oIfitzliTm2qoISfVacw3ETOAWvNT1irz0E6npjRrHJZttT/iL1aav9IPdcYDziE24BodcaX91XbquPG4eyVarXE0xZA7bch68IUxI7b0VKT31QocOvFPuJriupveyFfDslBWx3wNFYI9UO4l2Nfo+iiZfJc6IlwbRRHLrTSu8yHApsEyvaEalYLotlAh1zog+AcFmhVeA8zD6pfKoWocapwRbLh3gd60jhAk4yxk1ung54ez/Yo+/vspmkPaz/07WvssrJ+G22qHJ1SvhustkUrrSHxGLMN+3Rou1d7899p7nV/6CvpufPOclqqL8bqsazQWbqzFPGRaiwLRqFZu+CnakUmrTsXNTpr9O3ZSTev1qY5c55SuMvzSxoOvh9pCbEzAsKV/wb6cONqzavsGK0NDgDfCJd1fQcuLaIAN4RZYtn7spzTgPavODaUU/LJpHZvVxmMVplXMJi8O/LWl72hp5a1PRSmx9J8U1YLajpFQcS9eeiWH2vm6H0NiZsHYLhDD6LJovAsnE5nE3NQZfdrBMk/bzeWYRk4tehEaNnJuo9oauo7G2XcpCtGquJfdThwo+58f4/9Oplm5kTZbLlVGO6/pFx4+CjCzzaRsOeQzD4fx5WAF8ONTmP1u0SUGyEqT6JhyifL6y2suTJFMmbSncZx5rv/3mB3cARfkUzbWNBwhU8p1akMyX2rnabZfsLaDehbaNcAEzW04Uq4Wl3vHl/0cdOtzdp1G64uhORzVGY4T6wXlO2Y2Bl/UOcynTdgcFNSs2V6drKyDJZIWQVrK2TmGQjCWYTl2YWsWYlyrecqhEVsySf5La2h8n4krR6r1LpLzRF8RFe4LkeYnWkcqv32dDEJjAkSL8ka9EhrfhktOak3fkn4PFVKD47wXwTPcrwoTWtdUeurN6yeQbJRFWDhhOdw/QkVXYFNWnxKZIdZ3M1GzVj5XKHy9PdxWmIbXO+qptcW0LbXgNAvwzf5rVY0+rKlRHWr9v8LuXe5RNdZXqK9t1wmOKPyXeszGFR7aPaXtHQ57sxH8Dtv3Qwd/xyCKVvlSvxHWPUx/nqI5vOUUIKdGnUyazSmNNtolxNHho8P59LWdWtu6cMfewVE/mRDCKkjLmrylEOaT0+JD0qhR5nKv2LYpUcetr/gDVZOSdR/TsGqh0NzRQ/Z6Y3F6YpSq9dcuBDNkuKU03+wUAuhDKKhUaOS1NR0CYvPZIFSqNAL/42TF+3y3P/aOp1qTIkILo3hRQQy9KXykQxcj66N7gCmrxKx1mLaMfjqa+MEMzuVWKaEL7WA/1x8Q2T3YTgsxwJd4I+hwrDCaMZ/z/zftjZ597exiaCdemS/NcrWCIeyMpqEnBxmn44kxs/nPizywfb91cAxdmI3ATM3S6InzkskdtUgjJNh+e/jg63AZncdxSobEfHJrTvi5HJNALqc+zGFB7z8tZRJrICkQf1mLgk3z3NkzyWrLyT/9iL6ssiv3Z7yF9sqk4hBIEcLMtclwljC9AsQVTNTHLiDXaFK27ItmTaG25WpKSCe0ZwkPRd1S1bazj8B+2I0WFtq5WQ38RKxORD3Rcs0h3zu2LLoIDbE5ZMG9C6WgGPal4NjHrL85w82zeBUZTUMNGQm4P0fTEOo0Qqup6VrqZDq3e+5dex7ap4q8fZLvWVWJF3LeNeAsurbN9MAR4kStzOeIaUAa2HCAOtTEO/lkL7t8yfadRkN9QFROHZ5CYylwbDH6sVVK07XFJXQCPVS/Ibamp1ytg3n6SlREt3xZGGYL8XK64LkkJ8HMJcdlqRQkxeUuSPchzQogjEF2Vl/87E8+nYeckPzb9Fhux17OXEWkV6U1fIxmQeh3lW3mF8Q/gnQ8h6ysVWUhvQsJyUzfJA1Nob5ZLRULJ6XKG2mnteOQV4CTfMhz+j6fSPd5Bpnn7I3KttZnB+Z1qHXeGOM6OA7VjHBaPU92X3wtyj5nXewh7cRUpBPe58RP2Rsp2R/ywlaL2wkbmBP4098zrzM94XOiTQkTa3gOa/Rkzw8RM9W/GPM1+mVrE/tW2/dz2Y6hMUtpl5RT9Ecsnco/pRSTmZz4JxDd15aWY7I1DI3kENZo832NDg3TR0AILYogmtUm5N3WSsniFK6/QHs0/9i+v/+rCdtWFE7s7XaidindrVeKd3viH1JAaMPk4BMAHo4NCQ7ze25D4mOTg1WDN2HrjQ0hP1FqNfXveqM4tC3TSYZNNZFuFmUjPc6j4wgFmDNhv4Va2fhYFiDxMbH2OUCnwZX7TbLY/bZZ9nA9utM0yggWPgOx5XX2e+1S8xL6RlsnYJLPi73FU2lDlDbiFrL0XweC2v1ojNF2BtnX896c5p8QzdXid/7JME9N/IztOza29atipV18dZqviEZP4Bdsr0G+a2yGgGaa1MDKZ8eXWkPo4bVvSavy128gJEVO8qUr0t0h2hpiobeyRAaiTqHZrD62H/8+zrXNx3tlqTTSuUDiRTHt+HKiMnfvuTWkoOb2Ovm8vo/qOGcq62RWoB95GM1fDpS4nuJlS3HDN8dkKYuHENJUe1Orb4cuVY0VDk6dBnBP7Cl1iAs1Wjd+dvfETuhI4P8FjXS7lSsJoZcAAAAASUVORK5CYII=>
```

Comment:
Can't we have these images actually be files instead of base64?

Reply:
should be able to